### DISCUSSION TITLE
openwrt 版怎么看自己用的v及版本

### DISCUSSION THREAD

**Discussion by lizhiwei8061** [IrineSistiana/mosdns#650] [General]
openwrt 版怎么看自己用的v及版本

**Comment by qingtian110** [comment]
![image](https://user-images.githubusercontent.com/87856890/228202700-3e14f502-d6f8-4110-b15f-060ecf653677.png)
![image](https://user-images.githubusercontent.com/87856890/228203251-ea90d9f3-c50a-44e5-b0fb-f052bca753fd.png)

mosdns v5 功能真多
